### PR TITLE
Add improved docs

### PR DESCRIPTION
**Pull request by wooorm**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

[View rendered](https://github.com/remarkjs/remark-embed-images/blob/docs/readme.md)

Related-to: unifiedjs/unified#168.

/cc @nemo-omen @remcohaszing @ChristianMurphy @Murderlon @with-heart @joostdecock @mgan59

### CODE REVIEW

**Review by wooorm — COMMENTED** [review]
✅

File: index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {visit} from 'unist-util-visit'
 const relative = /^\.{1,2}\//
 
 /**
- * Plugin to embed local images as data URIs, inlining files as base64-encoded
+ * Plugin to embed local images as data URIs, inlining files as encoded (base64)
```
Comment (ChristianMurphy) [inline index.js]: parenthesis is a nice add, the original order seemed to flow a bit better, thoughts?
```suggestion
 * Plugin to embed local images as data URIs, inlining files as (base64) encoded
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "remark-embed-images",
   "version": "3.0.0",
-  "description": "remark plugin to embed images with data URIs, inlining base64-encoded sources",
+  "description": "remark plugin to embed local images as data URIs, inlining files as encoded (base64) values",
```
Comment (ChristianMurphy) [inline package.json]: similar with ordering
```suggestion
  "description": "remark plugin to embed local images as data URIs, inlining files as (base64) encoded values",
```

File: readme.md [hunk 3]
```diff
@@ -8,56 +8,96 @@
...
-[**remark**][remark] plugin to embed local images as data URIs, inlining files
-as base64-encoded values.
-
-## Note!
-
-This plugin is ready for the new parser in remark
-([`remarkjs/remark#536`](https://github.com/remarkjs/remark/pull/536)).
-No change is needed: it works exactly the same now as it did before!
+**[remark][]** plugin to embed local images as data URIs, inlining files
+as encoded (base64) values.
```
Comment (ChristianMurphy) [inline readme.md]:
```suggestion
as (base64) encoded values.
```

**Review by ChristianMurphy — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (remcohaszing) [reply index.js]: I’m personally not a fan of the added parentheres. I’d leave them out. I do like that the hyphen has been removed.

Comment (wooorm) [reply index.js]: Maybe the whole second phrase can be removed, as it’s implied by the earlier phrase already?

**Pull request merged**